_posts/2022-08-10-charis-blog1.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page 
+title: When Lennard-Jones Meets Monte Carlo  
+author: Charis Liao
+---
+
+# A Road Map to Measuring 
+There are different levels of measurement in Science. From macro level to micro level and even molecular level. In general physics classes, students usually measure different forces by throwing a ball and measuring the distance. However, it will be extremely difficult to use the same kind of measuring method in the molecular level. Therefore, we have to rely on molecular stimulations to calculate the statistical mechanics of the systems. 
+
+# Monte Carlo & Lennard-Jones Potential 
+
+Lennard-Jones Potential is a fantastic example of needing molecular stimulations to calculate the potential energy between two particles. One of the methods that has been widly used in computational science is the Monte Carlo method. It could be used to evaluate integrals (or evaluate an area under the curve). To apply the Monte Carlo method on a statistical mechanics model: 
+
+⟨Q⟩ = $\int_v$ Q($r^N$) $\rho$($r^N$)$dr^N$  
+
+Q = the thermodynamic quantity of interest (system potential energy)
+$\rho$($r^N$) = probability density  
+V = volume of configuration space
+
+Since the thermodynamic system (LJ Potential) of interest actually involves high dimensional integrals and are dominated by a relatively small region of configurational space, having **importantce sampling** and an **acceptance criteria** (Metropolis Criterion) could help generating sample coordinates from the desired equibilbrium probability density and a standard for accepting generated positions.  
+
+# Translating Words to Code - Tips and tricks to decrease the runtime of a Monte Carlo Stimulation  
+
+It is not surprised to implement multiple "for loops" to compare every single particle pair; however, having nested for loops will increase the runtime of our stimulation. At this point, we should ask ourselves, how can we avoid useless calculations and improve runtime? 
+
+A simple answer to the above question would be to only compare the particle of interested with other pariticles in the system instead of compare all possible particle pairs. This would lower the number of computation from n choose 2 permutations to n-1 times. 
+
+# RDF - The Radial Distribution Function 
+
+Theoretically, the distribution for pariticles should be the same; however, this is not possible because the attractions between molecules will hindered the uniform distribution of the configuration. This is where the radial distribution comes into play to analyze pair correlation of the local structure of particles. 
```

Comment:
I think globally, the particle distribution is indeed uniform.
It's only not uniform to the particle of interest at close distance. on global scale, the particle distribution is still uniform and you get g(r) = 1.